### ISSUE TITLE
Document "Codespaces: View Creation Log"

### ISSUE THREAD

**Issue by mchelen-gov** [MicrosoftDocs/vscodespaces#796]
There is a command "Codespaces: View Creation Log" which is helpful when troubleshooting devcontainer or startup logs. This command should be documented.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59c2b556-2e6c-7b4b-4fe8-01fd712130b7
* Version Independent ID: e8be4529-0bc7-af76-3c10-3989d49cffec
* Content: [Troubleshooting Visual Studio Codespaces - Visual Studio Codespaces](https://docs.microsoft.com/en-us/visualstudio/codespaces/resources/troubleshooting)
* Content Source: [vscodespaces/resources/troubleshooting.md](https://github.com/MicrosoftDocs/vscodespaces/blob/master/vscodespaces/resources/troubleshooting.md)
* Product: **visual-studio-family**
* Technology: **visual-studio-codespaces**
* GitHub Login: @nikmd23
* Microsoft Alias: **nimolnar**

**Comment by mchelen-gov** [comment]
also in terminal view -> Output -> VS Codespaces (different logs)

**Comment by MSFTJim** [comment]
I agree.  I just found out about it today and it saved me a ton of troubleshooting time!  I will be sharing broadly but would be beneficial to have it out in the docs, etc.
